docs/en/04_Changelogs/4.0.0.md

```diff
@@ -92,6 +93,138 @@ For a full list of renamed classes, check the `.upgrade.yml` definitions in each
 
 The rename won't affect class-based permission codes or database table names.
 
+##### Use ::class whenever possible
+
+Prefer:
+ ```php
+ private static $has_one = [
+   'MyObject' => MyObject::class
+ ];
+ ```
+ to
+ 
+ ```php
+private static $has_one = [
+  'MyObject' => 'My\Project\MyObject'
+];
+```
+
+##### Use Injector wherever you can
+
+This can help with reducing changes from 3.x to 4.x by allowing your code to 
+continue to use `MyClassName`, while you tell the injector what the FQ class name is:
+
+```yaml
+SilverStripe\Core\Injector\Injector:
+  MyClassName:
```

Comment:
This seems to be more of a "how to namespace your code" task than a "upgrade task". Upgrading to ss4 doesn't automatically namespace your code, nor is it required.

A better example is injector usage to construct DB Field types, eg. 


```diff
- $field = new Varchar();
+ $field = Injector::inst()->create('Varchar');
```